unstructured/file_utils/filetype.py

```diff
@@ -347,7 +368,17 @@ def _is_text_file_a_json(
         with open(filename) as f:
             file_text = f.read()
 
-    return re.match(LIST_OF_DICTS_PATTERN, file_text) is not None
+    lines = file_text.strip().splitlines()
+    if len(lines) < 2:
+        return False
+
+    header = lines[0].split(",")
+    columns = len(header) if len(header) < 5 else 5
```

Comment:
Is there a reason for making columns having a minimum length of 5?

Reply:
No. I wasn't thinking straight and was thinking that was the first 5 lines of the file (so we aren't iterating over the whole file). I'll change that.